Fix error when using Wyng with rclone

ldeso opened the pull request:
A simple and perhaps not-so-elegant solution to issue #175.

~~Edit: actually this doesn't fix the issue.~~
Edit 2: it does fix #175! :smiley: I forgot to apply the pull request on my local version of Wyng.

tasket reviewed (changes requested):
I changed the target branch to '08wip'.

You can make this fix more complete if you change the earlier 'untar' assignment line to include the '-tmp' extension like so:
`untar_cmd = [dest.cd + " && mkdir -p ./"+sdir+"-tmp`

If you prefer not to, just let me know and I'll do it.

ldeso commented:
Ah I see, thanks. I don't have time today but I can make and test the change tomorrow.

ldeso commented:
I rebased to the '08wip' branch and changed the 'untar' assignment, `wyng send` works.

Just to see what happens, I also tested a version of Wyng with only the 'untar' change and that didn't work: `wyng send` would hang indefinitely. `wyng send` would only work with a version of Wyng that has just the `rm -df` change, or that has both the `rm -df` change and the `untar` change.
